### ISSUE TITLE
Integration test for exposing bridge does not check things properly

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#1932]
After the latest changes on the master branch, the integration test for exposing Keptn Bridge does not work as it should.

```
$ test/utils/expose_keptn_bridge.sh
Exposing Keptn Bridge...
secret/bridge-credentials created
Error: please specify a user name for exposing the bridge using the '--user=<username>' flag
pod "bridge-7855b89b6d-5njtq" deleted
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
```

# Definition of Done

- [ ] The test needs to fail if the CLI command fails
- [ ] The test needs to fail if accessing Keptn Bridge does not work

**Comment by christian-kreuzberger-dtx** [comment]
This will be influenced by KEP 18, removing it from next-sprint.

**Comment by agrimmer** [comment]
This issue is obsolete because it was fixed in issue #1965
